### PR TITLE
Add GetJobRawLog API

### PR DESCRIPTION
**Pull request by Gerhut**
Need to match #1133 when it is merged

Response is successfully chunked per line, to validate

```
$ curl -v http://domain:5001/GetJobRawLog\?jobId\=id\&userName\=user@domain --raw
*   Trying ip...
* TCP_NODELAY set
* Connected to domain () port 5001 (#0)
> GET /GetJobRawLog?jobId=id&userName=user@domain HTTP/1.1
> Host: domain:5001
> User-Agent: curl/7.52.1
> Accept: */*
>
< HTTP/1.1 200 OK
< Date: Thu, 21 May 2020 09:10:59 GMT
< Server: Apache/2.4.18 (Ubuntu)
< Vary: Accept-Encoding
< Transfer-Encoding: chunked
< Content-Type: text/plain
<
4c
+ DLTS_RUNTIME_DIR=/dlts-runtime
+ cp -r /dlts-init/ssh_build /dlts-runtime

53
+ cp -r /dlts-init/ssh_config /dlts-runtime
+ cp /dlts-init/gpu_topo /dlts-runtime

2c
+ cp -r /dlts-init/install.sh /dlts-runtime

29
+ cp -r /dlts-init/runtime /dlts-runtime

25
+ python3 /dlts-init/runtime/sync.py

49
2020-05-15 00:56:50,123 - INFO - sync.py:187 - do not need to sync, skip

2f
+ chmod +x /dlts-runtime/status/job_command.sh

4a
+ runuser -s /bin/bash -l qixcheng -c /dlts-runtime/status/job_command.sh

0

* Curl_http_done: called premature == 0
* Connection #0 to host domain left intact
```

### CODE REVIEW

**Comment by xudifsd** [comment]
Please add more test case, either unit test or integration test.

Test cases can detect problem before deployment, and make sure future deployment won't break previous function.

File: src/ClusterManager/test/utils.py [hunk 1]
```diff
@@ -533,30 +533,16 @@ def block_until_state_in(rest_url, jid, states, timeout=300):
 
 
 def get_job_log(rest_url, email, jid):
```
[the remark this answers is not in the document]
Comment (Gerhut) [reply src/ClusterManager/test/utils.py:535]: Will deploy ES back to Dev2 cluster to do ES tests.

**Comment by Gerhut** [comment]
unit test (azure blob) done, based on #1133

**Comment by Gerhut** [comment]
QPS: 30
Max Concurrency: 120


Concurrent | Failed / 1k reqs | Time (ms) | Reqs / sec | Trans. Rate (KB / sec)
-- | -- | -- | -- | --
1 | 0 | 700 | 1.43 | 59
2 | 1 | 818 | 2.44 | 102
5 | 0 | 703 | 7.11 | 297
10 | 0 | 749 | 13.35 | 558
20 | 0 | 794 | 25.16 | 1052
30 | 0 | 1043 | 28.75 | 1202
40 | 0 | 1324 | 30.19 | 1262
50 | 0 | 1628 | 30.71 | 1284
100 | 0 | 3277 | 30.51 | 1276
110 | 0 | 3669 | 29.98 | 1254.08
120 | 0 | 3966 | 30.25 | 1265.45
130 | 5 | 4271 | 30.44 | 1266
140 | 162 | 3931 | 35.61 | 1249
200 | 650 | 2368 | 84.43 | 1268.53
500 | 147 | 2732 | 182.96 | 1217.53
1000 | 155 | 5752 | 173.85 | 1214.24

**Pull request merged**